specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2022-09-01-preview/definitions.json

```diff
@@ -8346,6 +8346,316 @@
         }
       },
       "description": "Wiki update contract details."
+    },
+    "WorkspaceCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WorkspaceContract"
+          },
+          "description": "Page values."
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total record count number across all pages."
+        },
```

Comment:
List API Response shouldn't have "count". It is not part of the ARM RPC (Resource Provider Contract) and will be unexpected by any generic ARM clients. You should not be adding count to any List Response.

See the public docs for ARM List Resources Response: [https://learn.microsoft.com/en-us/rest/api/resources/resources/list#resourcelistresult](https://learn.microsoft.com/en-us/rest/api/resources/resources/list#resourcelistresult).

Please apply to all such usage.